### DISCUSSION TITLE
Welcome to DevInterview_Questions Discussions!

### DISCUSSION THREAD

**Discussion by Tc2r1** [Tc2r1/Dev_Interview_Prep_App#30] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AndroidLion48** [comment]
This repository has been flagged for Hacktoberfest Exclusion on Oct 26.  
We are taking all steps possible to correct this error. 
If your willing;


Please send a email to:   hacktoberfest@digitalocean.com
With the subject line: 		False Flag/ Please Reinstate Repository

If you liked something, learned, practiced, taught someone, or challenged yourself; talk about your experience here.

Describe why you believe this project is in accordance with Hactoberfest Values (1), and GitHub Code of Conduct (2).

1.  https://github.blog/2022-10-04-expand-your-open-source-contributions-during-hacktoberfest-2022/
    https://hacktoberfest.com/about/#low-or-non-code
    https://hacktoberfest.com/participation/

2.  https://docs.github.com/en/site-policy/github-terms/github-community-code-of-conduct


Sincerely, Thank you for all your contributions to building this project!  Your contributions will teach someone essential software development knowledge.
